packages/slugify/slugify.d.ts

```diff
@@ -0,0 +1,21 @@
+declare const slugify: {
```

Comment:
This package must be written in Typescript, and then built properly if you want to publish it

Reply:
But I'm not sure we need this one

Reply:
removed